acropolis-persistence/src/main/kotlin/org/ephyra/acropolis/persistence/api/SystemSoftwarePersistence.kt

```diff
@@ -0,0 +1,16 @@
+package org.ephyra.acropolis.persistence.api
+
+import org.ephyra.acropolis.persistence.api.entity.SystemSoftwareEntity
+import org.ephyra.acropolis.persistence.impl.SystemSoftwareRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+@Component
+class SystemSoftwarePersistence {
+    @Autowired
+    private lateinit var repo: SystemSoftwareRepository
```

Comment:
Probably want to try and be consistent with naming this. Called `repo` in some places and `persistence` in others.